src/openvas.c

```diff
@@ -554,7 +554,8 @@ openvas (int argc, char *argv[], char *env[])
     }
   tzset ();
 
-  if ((err = init_logging ()) != 0)
+  err = init_logging ();
+  if (err != 0)
```

Comment:
```suggestion
  if (init_logging () != 0)
```
we do not need to save the err here, as we overwrite it anyways